ci/jenkins/test.sh

```diff
@@ -380,24 +380,18 @@ function deliver_antrea_linux {
 
     echo "==== Start building and delivering Linux Docker images ===="
     DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
-    if [[ "$TESTCASE" == "windows-networkpolicy-process" ]]; then
-        make windows-bin
-    fi
 
     echo "====== Delivering Antrea to all Nodes ======"
     export_govc_env_var
 
-    # Enable verbose log for troubleshooting.
-    sed -i "s/--v=0/--v=4/g" build/yamls/antrea-windows.yml
-
-    if [[ "${PROXY_ALL}" == false && ${TESTCASE} =~ "windows-e2e" ]]; then
-        sed -i "s|.*proxyAll: true|      proxyAll: false|g" build/yamls/antrea.yml build/yamls/antrea-windows.yml
+    if [[ "${PROXY_ALL}" == false ]]; then
+        sed -i "s|.*proxyAll: true|      proxyAll: false|g" build/yamls/antrea.yml
```

Comment:
do we even need this case now, given that `proxyAll: false` is the default for Linux?